src/Helpers/finalizeProcessing.js

```diff
@@ -150,8 +150,9 @@ module.exports = function finalizeProcessing(Processing, options) {
    * source and bind to canvas via new Processing(canvas, sourcestring).
    * @param {CANVAS} canvas The html canvas element to bind to
    * @param {String[]} source The array of files that must be loaded
+   * @param {Function} complete A callback, called with the sketch as the argument.
    */
-  var loadSketchFromSources = Processing.loadSketchFromSources = function(canvas, sources) {
+  var loadSketchFromSources = Processing.loadSketchFromSources = function(canvas, sources, complete) {
```

Comment:
small nit: let's call this `onComplete` internally, to highlight that it's a callback handler.